# Model for the DriX

# Engine: 38HP  Nanni N4.38
# Power at crankshaft 27.6 kW (37.5hp)
# Rated speed 3000 rpm
# idle speed 850 rpm
# Peak torque 96 Nm at 2200 rpm

# Transmission ZF25A
# ratio? guessing about 2.4? 1/2.4 = 0.417

# Loaded weight: 1600kg (from specs)
# measured weight: 1900kg
# measured weight with water in gondola: 2200kg

# Speed: min 2-5 knots/ max 14 knots
# RPM Range Forward: 1000 to 3200
#     Reverse: 800 to 1200
# No reverse in autopilot mode
# RPM under 1200 to switch between fwd, neutral or rev

models/drix:
  max_rpm: 3200
  max_power: 27600
  idle_rpm: 850
  clutch_engagement_rpm:  800
  gearbox_ratio:          0.417
  propulsion_type: prop
  prop_pitch: 16
  max_rpm_change_rate: 1000
  max_speed: 7.0
  mass: 2200

  rudder_distance:        3.4

  max_rudder_angle:       30
  max_rudder_change_rate: 90
  rudder_coefficient:   .25
